app/controllers/api/journeys_controller.rb

```diff
@@ -145,6 +145,7 @@ def create_event
         notes: 'Automatically generated event',
         details: {},
         supplier_id: supplier.id,
+        created_by: created_by,
```

Comment:
could you add a test in the journey's controller create/update specs as well, as the ones in the journeys **events** controller are inheriting from the eventable concern